components/tabs/WithdrawTab.tsx

```
@@ -0,0 +1,538 @@
+
+              case "success":
+                updatedProgress.steps[1].status = "success";
+                updatedProgress.currentStepIndex = 2;
+                updatedProgress.steps[2].status = "success";
+                updatedProgress.overallStatus = "success";
+                break;
+
+              case "error":
+                const processingIndex = updatedProgress.steps.findIndex(
+                  (step) => step.status === "processing",
+                );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)